core/application/app_test.go

```diff
@@ -409,11 +409,18 @@ func TestApp_GetAllMicroVM(t *testing.T) {
 		expect      func(rm *mock.MockMicroVMRepositoryMockRecorder, em *mock.MockEventServiceMockRecorder, im *mock.MockIDServiceMockRecorder, pm *mock.MockMicroVMServiceMockRecorder)
 	}{
 		{
-			name:        "empty namespace should return an error",
+			name:        "empty namespace should not return an error",
```

Comment:
do we want equivalent test for name?

Reply:
A lot of them are testing that. And actually there is not test that receives the name.